### ISSUE TITLE
32-bit images

### ISSUE THREAD

**Issue by luc-j-bourhis** [CentOS/sig-cloud-instance-images#51]
It would be nice to have i386 images for CentOS 5 and 6 from this official channel.

**Comment by staticfloat** [comment]
I second this, it would be really nice to have i386 images.  As it stands, there aren't many resources for 32-bit images on Docker, and CentOS has great 32-bit support.

**Comment by dsandstrom** [comment]
@staticfloat I see that you added one https://hub.docker.com/r/staticfloat/centos-i386/
Also, there is this one https://hub.docker.com/r/toopher/centos-i386/

Both give me errors when trying to use yum, so would love an official version that works.

**Comment by staticfloat** [comment]
What errors are you having?   My image is specifically designed to allow for `yum` usage when running on an `overlayfs2` storage driver.  Mine is a fork of the `toopher/centos-i386` version, I have a PR open against that repository trying to upstream my changes: https://github.com/toopher/toopher-docker/pull/6

**Comment by dsandstrom** [comment]
Basically this https://github.com/CentOS/sig-cloud-instance-images/issues/15, but with it complaining about the checksum for curl.

The suggestion is to install `yum-plugin-ovl` first like your PR, but that didn't seem to fix it.  And other people say that the docker base already has it included.

Using the official 64 bit version works, but I think I need to compile my app in 32 bit environment.

**Comment by staticfloat** [comment]
You should be able to verify that `yum-plugin-ovl` is already installed with `yum list installed`:

```
# yum list installed | grep yum-plugin-ovl
yum-plugin-ovl.noarch                1.1.30-40.el6                      @base   
```

It looks like the `staticfloat/centos-i386:6` image doesn't have it installed, but the `staticfloat/centos-i386:centos6` image does.  I'll fix that; that's probably where your issue is coming from.

**Comment by dsandstrom** [comment]
Indeed, I tried toopher:centos6 and your centos-i386:6.  However, your centos-i386:centos6 is working better.  Thank you for the help and the image.  Do you have up on github?  It appears toopher's is abandoned.

**Comment by staticfloat** [comment]
Yes, it's linked from my pull request above, and is also available [here](https://github.com/staticfloat/toopher-docker/tree/sf/yum_plugin).

**Comment by dsandstrom** [comment]
You might think about allowing issues if you plan to support the fork.

**Comment by staticfloat** [comment]
I'm not really looking to support it, sorry.  I use it to build my own images on top of once every few months, but if you ping me on a related issue like this, I may respond if I have the time.  No guarantees of support though, as I already have too many other things going on.
